modules/microshift-nw-router-config-IPs.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-ip-addresses_{context}"]
+= Configuring router IP addresses
+
+You can restrict the network traffic to the router by configuring specific IP addresses. For example:
+
+* Use cases where the router is reachable only on internal networks, but not on northbound public networks.
+* Use cases where the router is reachable only by northbound public networks, but not on internal networks.
+* Use cases where the router is reachable by both internal networks and northbound public networks, but on separate IP addresses.
+
+.Prerequisites
+
+* You installed {microshift-short}.
+* You created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the IP addresses in the `ingress.listenAddress` field in the {microshift-short} `config.yaml` as shown in the following example:
++
+.Example router IP address settings
+[source,yaml]
+----
+#....
+ingress:
+  listenAddress:
+    - "" # <1>
+# ....
+----
+<1> The `ingress.listenAddress` value defaults to the entire network of the host. Valid customizable values can be a single IP address or host name or a list of IP addresses or host names. To use the default setting, remove the `listen.Address` field.
++
+[IMPORTANT]
+====
+You must either remove the `listenAddress` parameter or add values to it when using the `config.yaml` file. Do not leave the field empty or {microshift-short} crashes on restart.
+====
++
+[TIP]
+====
+If you complete all the configurations that you need to make in the {microshift-short} `config.yaml` file at the same time, you can minimize system restarts.
+====
+
+//.Verification
+//TODO Yes, Pablo to provide example; see end-to-end tests
```

Comment:
Tried to come up with a simple example, but it is still too complex for the docs. It implies configuring MicroShift and using tools like curl from within/outside the MicroShift host. Can we skip it?

Reply:
just make sure the the listen addresses(load balancer IPs) are reachable, the you can curl the route with the destination to a load balancer ip. I think we can skip it.

Reply:
I just gave your basic description, @ShudiLi, if you think is OK.